zstd-safe/zstd-sys/build.rs

```diff
@@ -133,21 +136,23 @@ fn compile_zstd() {
 
 fn main() {
     // println!("cargo:rustc-link-lib=zstd");
-
-    if !PathBuf::from("zstd/lib").exists() {
-        panic!("Folder 'zstd/lib' does not exists. Maybe you forget clone 'zstd' submodule?");
-    }
-
     let (defs, headerpaths) = if cfg!(feature = "pkg-config") {
         pkg_config()
     } else {
         if !PathBuf::from("zstd/lib").exists() {
             panic!("Folder 'zstd/lib' does not exists. Maybe you forget clone 'zstd' submodule?");
         }
 
+        let manifest_dir = PathBuf::from(
+            env::var("CARGO_MANIFEST_DIR")
+                .expect("Manifest dir is always set by cargo"),
+        );
+
         compile_zstd();
-        (vec![], vec![PathBuf::from("zstd/lib")])
+        (vec![], vec![manifest_dir.join("zstd/lib")])
     };
 
+    println!("cargo:include={}", headerpaths.iter().map(|p| p.display().to_string()).join(";"));
```

Comment:
Do we need `Display` for that? Why not just `iter().filter_map(PathBuf::to_str)` or something like that?

Reply:
But doesn't that mean we could loose some parts? The documentation says that the display mitigates non Unicode paths.

Reply:
True, but if a path contains non-unicode parts, will this actually work as an include path? I would think the "unicode-safety" here means a lossy conversion that may work for pretty-printing, but might not work as an include path.